core/src/main/java/com/opensymphony/xwork2/ognl/OgnlValueStack.java

```diff
@@ -204,6 +204,9 @@ protected void handleRuntimeException(String expr, Object value, boolean throwEx
     }
 
     protected void handleOgnlException(String expr, Object value, boolean throwExceptionOnFailure, OgnlException e) {
```

Comment:
Null-guard:
` if (e != null && e.getReason() instanceof SecurityException) {`
so that this change doesn't risk creating an NPE where one couldn't happen before if the passed `e` parameter is null.